cli/tests/041_info_flag.out

```diff
@@ -2,3 +2,4 @@ DENO_DIR location: "[WILDCARD]"
 Remote modules cache: "[WILDCARD]deps"
 Emitted modules cache: "[WILDCARD]gen"
 Language server registries cache: "[WILDCARD]registries"
+Origin data location: "[WILDCARD]location_data"
```

Comment:
This should be individual paths to each DB file IMO (assuming there'll be another for indexeddb in the future)
```suggestion
Origin local storage: "[WILDCARD]local_storage"
```

Reply:
`local_storage` is a file in `location_data` under a hashed `--location`. indexeddb will be another file in the same dir, and same for other APIs that will require to have files on the system based on the origin (ie broadcast channel IPC)

Reply:
Yes it was a mistake to highlight this test. What will the output for `deno info --location https://foo/` be? I think it should be a direct path to the local storage db file, if it isn't already:
```
Local storage: /home/nayeem/.cache/deno/location_data/<hash>/local_storage
IndexedDB: /home/nayeem/.cache/deno/location_data/<hash>/indexed_db
```

Reply:
Currently it is
```
$  deno info --location=https://deno.land
Origin data location: "/Users/crowlkats/Library/Caches/deno/location_data/491d1c5f8d5eaa49d0ee3166c5aca7fbf9af5c466365713577ab3c739342712c"
```
I dont think we should have an entry for each API that will need to use persistent data based on the origin

Reply:
It's more like what we do with the other cache files. We should be as specific as possible and only stop where the structure within a directory is free-form, otherwise we are withholding info. It's just as unneeded as the 2nd and 3rd fields in:
```
DENO_DIR location: "/home/nayeem/.cache/deno"
Remote modules cache: "/home/nayeem/.cache/deno/deps"
Emitted modules cache: "/home/nayeem/.cache/deno/gen"
Language server registries cache: "/home/nayeem/.cache/deno/registries"
```

Reply:
Actually in that vein we should have the following fields:
```
Web storage: /home/nayeem/.cache/deno/location_data/<hash>
Local storage: /home/nayeem/.cache/deno/location_data/<hash>/local_storage
IndexedDB: /home/nayeem/.cache/deno/location_data/<hash>/indexed_db
```
It allows for more concise and less assuming consumption.

Reply:
> `Web storage: /home/nayeem/.cache/deno/location_data/<hash>`

why `Web storage`?

Reply:
The field is there same as `DENO_DIR location` for easy deletion. Or the name? `Origin data` would be more accurate for the broadcast channel stuff, that's fine.

Reply:
@nayeemrmn PTAL